CHANGELOG.md

```diff
@@ -18,9 +18,7 @@
 * Added a speed limit indicator to the upper-left corner of the map view during turn-by-turn navigation (upper-right corner in CarPlay). To hide the speed limit indicator, set the `NavigationViewController.showsSpeedLimits` property to `false`. To customize the indicator’s colors, configure `SpeedLimitView`’s appearance proxy inside a `Style` subclass. ([#2291](https://github.com/mapbox/mapbox-navigation-ios/pull/2291))
 
 ### Spoken instructions
-* Removed `MapboxVoiceController.play(_:)` in favor of `MapboxVoiceController.play(instruction:data:)`. ([#2230](https://github.com/mapbox/mapbox-navigation-ios/pull/2230), [#2297](https://github.com/mapbox/mapbox-navigation-ios/pull/2297))
-* The `MapboxVoiceController.speakWithDefaultSpeechSynthesizer(_:error:)` and `VoiceControllerDelegate.voiceController(_:spokenInstructionsDidFailWith:)` methods now accept a `SpeechError` instance instead of an `NSError` object. ([#2230](https://github.com/mapbox/mapbox-navigation-ios/pull/2230))
-* Added the `VoiceControllerDelegate.voiceController(_:didFallBackTo:becauseOf:)` method for detecting when the voice controller falls back to `AVSpeechSynthesizer`. ([#2230](https://github.com/mapbox/mapbox-navigation-ios/pull/2230))
+* Introduced `SpeechSynthesizing` for implementing custom Speech Synthesizing engines, as well as built-in implementations `SystemSpeechSynthesizer` and `MapboxSpeechSynthesizer` for system SpeechSynthesizer and Mapbox Speech engine respectively. This splits `RouteVoiceController` functionality into separate parts. `MapboxVoiceController` is replaced with alternative `SpeechSynthesizersController` which allows multiple Synthesizers with a fallback mechanism. ([#2268(https://github.com/mapbox/mapbox-navigation-ios/pull/2348)])
```

Comment:
```suggestion
* Replaced `RouteVoiceController` and `MapboxVoiceController` with `SpeechSynthesizerController`. `SpeechSynthesizerController` coordinates multiple cascading speech synthesizers. By default, the controller still tries to speak instructions via the Mapbox Voice API (`MapboxSpeechSynthesizer`) before falling back to VoiceOver (`SystemSpeechSynthesizer`), but you can also provide your own speech synthesizer that conforms to the `SpeechSynthesizing` protocol. ([#2268](https://github.com/mapbox/mapbox-navigation-ios/pull/2348))
```

I realize RouteVoiceController technically still exists, but it’s no longer as interesting to the application developer as something that can be swapped out; it’s more likely that the developer would want to swap out individual speech synthesizers.